easybuild/easyconfigs/b/BLIS/BLIS-0.8.1_fix_dgemm-fpe-signalling-on-broadwell.patch

```diff
@@ -0,0 +1,2219 @@
+Taken from https://github.com/flame/blis/pull/544
+Fixes a problem with DGEMM causing FPR signalling on Broadwell
+See https://github.com/flame/blis/pull/544
```

Comment:
Do we need to refer to the PR twice?

I guess you intended to refer to https://github.com/flame/blis/issues/486 here?